tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/ControlSyntaxSuite.scala

```diff
@@ -1370,6 +1370,114 @@ class ControlSyntaxSuite extends BaseDottySuite {
     )
   }
 
+  test("match-chained") {
+    runTestAssert[Stat](
+      """|xs match {
+         |  case Nil => "empty"
+         |  case x :: xs1 => "nonempty"
+         |} match {
+         |  case "empty" => 0
+         |  case "nonempty" => 1
+         |}
+         |""".stripMargin,
+      assertLayout = None
```

Comment:
It will not be pretty but we could add expected layout. but also okay not to add :)

Reply:
I added to the other cases.